proxy/contracts/schain/CommunityLocker.sol

```diff
@@ -93,6 +98,10 @@ contract CommunityLocker is ICommunityLocker, AccessControlEnumerableUpgradeable
 
     uint256 public gasPriceTimestamp;
 
+    mapping(bytes32 => uint) public timeLimitPerMessage;
+
+    mapping(bytes32 => mapping(address => uint)) public lastMessageTimeStampToSchain;
```

Comment:
Why adding of `lastMessageTimeStampToSchain` does not deprecate `lastMessageTimeStamp`?

Reply:
It does

Reply:
OK.